docs/jsonapi-spec.md

```diff
@@ -536,14 +536,93 @@ Creates a new collection in the current namespace.
 ```json
 {
   "createCollection": {
-    "name": "purchase"
+    "name": "some_collection",
+    "options" : {
+      "vector" : {
+      "dimension" : 5,
```

Comment:
Make sure to indent 2 spaces, otherwise it looks like `dimension` is a sibling of `vector` rather than a child